inference-engine/src/inference_engine/precision_utils.cpp

```diff
@@ -129,7 +129,7 @@ ie_fp16 f32tof16(float x) {
         if (v.u & 0x007FFFFF) {
             return s | (v.u >> (23 - 10)) | 0x0200;  // return NAN f16
         } else {
-            return s | (v.u >> (23 - 10));  // return INF f16
+            return s | EXP_MASK_F16;  // return INF f16
```

Comment:
we also have `f32tof16` inside VPU perf check. Does it contain the issue? The implementations look different..

Reply:
Excellent catch!
Yes, it actually has the same issue. I still don't get why perfcheck has these implementations inside, while it may just reuse them from the common part (maybe there were some intention to introduce as less external dependencies as we can, but `plugin_api` is linked anyway).
So I've just removed this dead code and reused the fixed one.